docs/configuration.md

```diff
@@ -108,7 +108,8 @@ Sugoi-api implements spring security with full customization allowed
 | fr.insee.sugoi.api.old.regexp.role.admin                   |                                                                                                                        |               |         |
 | fr.insee.sugoi.api.old.enable.preauthorize                 |                                                                                                                        |               |         |
 | fr.insee.sugoi.api.regexp.role.reader                      |                                                                                                                        |               |         |
-| fr.insee.sugoi.api.regexp.role.writer                      |                                                                                                                        |               |         |
+| fr.insee.sugoi.api.regexp.role.writer 
+| fr.insee.sugoi.api.regexp.role.admin.realm                     |                                                                                                                        |               |         |
```

Comment:
je propose : | fr.insee.sugoi.api.regexp.role.admin.realm                     |      Default pattern to use when searching the admin of a realm or a userstorage.  Realm should be passed as $(realm) and for a userstorage admin userstorage is passed as $(userstorage)     |     no default     |    ROLE\_SUGOI\_\$(realm)\_ADMIN,ROLE\_SUGOI\_\$(realm)\_\$(userStorage)\_ADMIN     |